openslides_backend/action/user/set_password.py

```diff
@@ -0,0 +1,33 @@
+from typing import Any, Dict
+
+from ...models.models import User
+from ..default_schema import DefaultSchema
+from ..generics import UpdateAction
+from ..register import register_action
+
+
+@register_action("user.set_password")
+class UserSetPasswordAction(UpdateAction):
+    """
+    Action to set the password and default_pasword.
+    """
+
+    model = User()
+    schema = DefaultSchema(User()).get_update_schema(
+        required_properties=["password"],
+        additional_optional_fields={"set_as_default": {"type": "boolean"}},
+    )
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        set hashed password and set default password if set_as_default is True.
+        """
+        password = instance.pop("password")
+        set_as_default = False
+        if "set_as_default" in instance:
+            set_as_default = instance.pop("set_as_default")
+        hashed_password = self.auth.hash(password)
+        instance["password"] = hashed_password
+        if set_as_default:
+            instance["default_password"] = hashed_password
```

Comment:
`default_password` must contain the password in clear text

Reply:
Done.